### PR TITLE
[PATCH API-NEXT v1] Fixing typos apinext

### PR DESCRIPTION
**Pull request by MykytaI**

### CODE REVIEW

File: include/odp/api/spec/traffic_mngr.h [hunk 1]
```diff
@@ -97,7 +97,7 @@ extern "C" {
 
 /**
  * @def ODP_TM_MIN_SHAPER_BW
- * The largest amount of bandwidth that any shaper's peak or commit rate can
+ * The lowest amount of bandwidth that any shaper's peak or commit rate can
```
Comment (Bill-Fischofer-Linaro) [inline include/odp/api/spec/traffic_mngr.h]: @psavol Please sign off on this change since it's technically not a typo fix.

**Comment by muvarov** [comment]
Merged.

**Comment by muvarov** [comment]
uh, merging patches in the morning,  skipped Petri's comment.

**Comment by muvarov** [comment]
@psavol  thanks!

**Pull request closed without merging**